README.md

```diff
@@ -1 +1,62 @@
-# Ylitse mobile application
+<h1 align="center"> Ylites Mobile App</h1> <br>
+<p align="center">
+    <img alt="Ylitse" title="Ylitse" src="https://github.com/ylitse/ylitse-app/blob/master/ylitse_logo.png">
+</p>
+
+<p align="center">
+  <a href="#">
+    <img alt="Download on the App Store" title="App Store" src="http://i.imgur.com/0n2zqHD.png" width="140">
+  </a>
+
+  <a href="#">
+    <img alt="Get it on Google Play" title="Google Play" src="http://i.imgur.com/mtGRPuM.png" width="140">
+  </a>
+</p>
```

Comment:
wow some html stuff and images!

Reply:
yeapp : D Only some xml in your markdown ¯\_( ͡° ͜ʖ ͡°)_/¯